resources/knative-serving/templates/istio-knative-extras.yaml

```diff
@@ -0,0 +1,352 @@
+---
+# Source: istio/charts/gateways/templates/serviceaccount.yaml
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cluster-local-gateway-service-account
+  namespace: istio-system
+  labels:
+    app: cluster-local-gateway
+    chart: gateways
+    heritage: Tiller
+    release: RELEASE-NAME
+---
+
+
+---
+# Source: istio/templates/serviceaccount.yaml
+# NOTE: excluded because already part of the "istio" component
+#apiVersion: v1
+#kind: ServiceAccount
+#metadata:
+#  name: istio-multi
+#  namespace: istio-system
+
+---
+# Source: istio/templates/clusterrole.yaml
+# NOTE: excluded because already part of the "istio" component
+#kind: ClusterRole
+#apiVersion: rbac.authorization.k8s.io/v1
+#metadata:
+#  name: istio-reader
+#rules:
+#  - apiGroups: ['']
+#    resources: ['nodes', 'pods', 'services', 'endpoints', "replicationcontrollers"]
+#    verbs: ['get', 'watch', 'list']
+#  - apiGroups: ["extensions", "apps"]
+#    resources: ["replicasets"]
+#    verbs: ["get", "list", "watch"]
+
+---
+# Source: istio/templates/clusterrolebinding.yaml
+# NOTE: excluded because already part of the "istio" component
+#apiVersion: rbac.authorization.k8s.io/v1
+#kind: ClusterRoleBinding
+#metadata:
+#  name: istio-multi
+#  labels:
+#    chart: istio-1.3.5
+#roleRef:
+#  apiGroup: rbac.authorization.k8s.io
+#  kind: ClusterRole
+#  name: istio-reader
+#subjects:
+#- kind: ServiceAccount
+#  name: istio-multi
+#  namespace: istio-system
+
+---
+# Source: istio/charts/gateways/templates/service.yaml
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: cluster-local-gateway
+  namespace: istio-system
+  annotations:
+  labels:
+    chart: gateways
+    heritage: Tiller
+    release: RELEASE-NAME
```

Comment:
this looks a bit odd, is it our release? should we put a release there?

Reply:
The file was copied unchanged from the file referenced in the README. I could change that, is that what we usually do when pulling stuff from Knative?

Reply:
tbh: idk

Reply:
Those are actually leftovers from `helm template`, which is used by the Knative folks to generate this file.
Nice catch 👍